examples/basic_mysql_connection/index2.js

```diff
@@ -0,0 +1,30 @@
+var fs = require('fs');
+var mysql = require('mysql');
+
+var config = JSON.parse(fs.readFileSync("config-secret.json"))
+
+var connection = mysql.createConnection({
+		host: config.host,
+		user: config.user,
+		password: config.password,
+		port: config.port,
+		database: config.database
+});
+
+
+const sqlStatments=[
+				'INSERT INTO todos (Name, Done) VALUES ("Odai",1)',
+				'UPDATE todos SET Name = "Odai" WHERE Id=20',
+				`UPDATE todos SET StatusId=2  WHERE Name="odai"`, 
+				'DELETE FROM todos WHERE Name="odai"',

```

Comment:
Notice that your name on line 17 is "Odai", but here on line 19, you use "odai". I actually had to test this to figure out what it would do, because I wasn't sure - looks like MySQL ignores case-sensitivity even in string comparisons - but this is REALLY not something I would rely on.

If you want to delete "Odai", ask to delete "Odai". Don't ask to delete "odai" and hope that it does the right thing - this time it did, but in many other similar cases in programming, it won't.